tutorials/notebooks/example_keras_effdet_lite0.ipynb

```diff
@@ -46,7 +46,7 @@
    "outputs": [],
    "source": [
     "!pip install -q tensorflow\n",
-    "!pip install -q model-compression-toolkit\n",
+    "!pip install -q mct-nightly\n",
```

Comment:
General regarding the code: shorten long lines (because it is noticeable on colab)